tito.spec

```diff
@@ -1,20 +1,20 @@
 %if 0%{?fedora}
 %global use_python3 1
 %global use_python2 0
-%global pythonbin %{__python3}
-%global python_sitelib %{python3_sitelib}
+%global ourpythonbin %{__python3}
+%global our_sitelib %{python3_sitelib}
 %else
 %global use_python3 0
 %global use_python2 1
 %if 0%{?__python2:1}
-%global pythonbin %{__python2}
-%global python_sitelib %{python2_sitelib}
+%global ourpythonbin %{__python2}
+%global our_sitelib %{python2_sitelib}
 %else
-%global pythonbin %{__python}
-%global python_sitelib %{python_sitelib}
+%global ourpythonbin %{__python}
+%global our_sitelib %{our_sitelib}
```

Comment:
All this goop can be dropped for a much simpler structure:

```specfile
%if (0%{?rhel} && 0%{?rhel} < 8)
# RHEL < 8 has no Python 3
%global __python %{__python2}
%else
# Default to Python 3
%global __python %{__python3}
%endif
```

From there, `%python_sitelib`, `%py_build`, `%py_install`, and so on have the desired effect.